Ran "Redundant Rule Checker" on AdGuard Base

DandelionSprout opened the pull request:
[//]: # (***You can delete or ignore strings starting with "[//]:" They will not be visible either way.)

***Description***:
* **Current behaviour**: 

[//]: # (Substitute this line with a description of the problem)

I figured that since "Hacktober" has begun, and it'd probably seem underwhelming for me to just do a "Fix-PR for one or two issue reports" thing, I thought it'd seem more impressive to do redundancy removal PRs like these.

My plan was to go through the 178 most important lines I got in the report when I ran AdGuard Base as a whole through https://abp.surge.sh/redundantRuleChecker/. However, I began to lose track of AdGuard Base's 15 source files after a little more than an hour, and Fanboy seemed to me personally to cast doubt about how desired it was to remove redundant rules in major lists when [I did the same in EasyList](https://github.com/easylist/easylist/pull/6104), so I only went through ~140 of the 178 report lines.

```
/:\/\/rarbg.bypassed.org\/cdn\/static\/20\/js\/[a-z0-9]{20,}.js/$domain=rarbg.bypassed.org has been made redundant by /:\/\/rarbg.bypassed.org\/cdn\/static\/20\/js\/[a-z0-9]{20,}.js/$domain=rarbg.bypassed.org
@@/js/showad_.js$domain=flashx.tv has been made redundant by @@/js/showad*.js$domain=flashx.tv|flashx.to
@@|https:///ovh.webshark.pl/adserver/*/main_script.js?advertise_check=1 has been made redundant by @@||ovh.webshark.pl/adserver/*/main_script.js?advertise_check=1
@@||adm.fwmrm.net/p/msn_live/AdManager.swf$domain=msn.com has been made redundant by @@||adm.fwmrm.net/p/msn_live/AdManager.swf$domain=msn.com|akamaized.net
@@||ajax.googleapis.com/ajax/libs/jquery/$domain=2ddl.unblocked.vc|daporn.com|imagefap.com|linkdrop.net|gelbooru.com|fmovies.is|fmovies.se|imagespicy.site|mycloud.to has been made redundant by @@||ajax.googleapis.com/ajax/libs/jquery/
@@||androgalaxy.in^$generichide has been made redundant by @@||androgalaxy.in^$generichide
@@||aphim.co^$generichide has been made redundant by @@||aphim.co^$generichide
@@||apitvh.net/embed/ads_code_1.js$domain=apitvh.net has been made redundant by @@||apitvh.net/embed/*ads_code_*.js
@@||asianovel.com^$generichide has been made redundant by @@||asianovel.com^$generichide
@@||bitearns.com/*/libs/advertisement.js has been made redundant by @@||bitearns.com/*/advertisement.js
@@||cdn.staticdata.site/js/loadjs.min.js^$domain=strikeout.nu has been made redundant by @@||cdn.staticdata.site/js/loadjs*.min.js$domain=strikeout.nu
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js$domain=ac-mo.com|nfmovies.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js$domain=asahihen.blogspot.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js$domain=mp4mania.xyz has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js$domain=she2013.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js$domain=sport-tv-guide.live has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js$domain=streamcloud.eu has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js$domain=televisiongratishd.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js$domain=todosobrepsp.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js$domain=ventax.net has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=1004lucifer.blogspot.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=abazero.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=androidaba.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=aphim.co has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=c11.kr has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=comicspace.com.br has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=daemon-hentai.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=donia2link.net|sqrible.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=evilkingmedia.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=ezbit.co.in has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=faucetcrypto.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=filmninja.ws has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=getpaidtoshare.money has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=giga74.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=gmyankee.tistory.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=highresolutionmusic.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=hotanime.me has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=jyra.online has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=link4gen.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=loadvid.online|hdonline.is has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=marimo.me has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=mega-hentai2.blogspot.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=memoryhackers.org has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=naszraciborz.pl has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=nekopoi.ga has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=no1lyrics.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=o2tvseries.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=okcaller.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=onlineinterviewquestions.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=orangespotlight.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=ourl.io has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=papalah.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=pitbullofficial.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=premium4ever.blogspot.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=proapkmod.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=sadistic.pl has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=satoshihero.com|satoshimonster.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=shiroyasha.me has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=tekloggers.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=televisiongratisenvivo.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=tv-onlinehd.com has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=uiz.io|uiz.app has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=upost.info has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=vcstream.to has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=voxc.org has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=wikiall.org has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=wikimotors.net has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/3.2.1/fuckadblock.min.js$domain=yungeblog.xyz has been made redundant by @@||cdnjs.cloudflare.com/ajax/libs/fuckadblock/*/fuckadblock.min.js
@@||facebook.com/adsmanager/manage^ has been made redundant by @@||facebook.com/adsmanager^
@@||getfree.co.in/*/libs/advertisement.js has been made redundant by @@||getfree.co.in/*/libs/advertisement.js
@@||imgadult.com/anex/alt2.js has been made redundant by @@||imgadult.com/anex/alt2.js
@@||kissasian.sh/Scripts/adsense.js has been made redundant by @@||kissasian.sh/Scripts/*.js
@@||mediaite.com/adbait/adsbygoogle.js has been made redundant by @@||mediaite.com/*/adsbygoogle.js
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=receive-sms-online.info has been made redundant by @@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=receive-sms-online.info
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=repo.xarold.com has been made redundant by @@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=repo.xarold.com
@@||pagead2.googlesyndication.com/pagead/js/r*/show_ads_impl.js$domain=lolalytics.com has been made redundant by @@||pagead2.googlesyndication.com/pagead/js/*/show_ads_impl.js$domain=lolalytics.com
@@||partner.googleadservices.com/gpt/pubads_impl_*.js$script,domain=indiatimes.com has been made redundant by @@||partner.googleadservices.com/gpt/pubads_impl_$domain=indiatimes.com
@@||singingdalong.blogspot.com^$generichide has been made redundant by @@||singingdalong.blogspot.com^$generichide
@@||static.clmbtech.com/ad/commons/js/colombia_v2.js$script,domain=indiatimes.com has been made redundant by @@||static.clmbtech.com/ad/commons/js/colombia_v2.js$domain=indiatimes.com
@@||stats.g.doubleclick.net/*.$xmlhttprequest,domain=uploadship.com has been made redundant by @@||doubleclick.net/*.$xmlhttprequest,domain=uploadship.com
@@||uploadbank.com^$generichide has been made redundant by @@||uploadbank.com^$generichide
@@||uploadship.com/themes/pro/frontend_assets/js/advertisement.js has been made redundant by @@||uploadship.com/*/*adv*.js
@@||winhelp.us^$generichide has been made redundant by @@||winhelp.us^$generichide
@@||infosehatku.club^$generichide has been made redundant by @@||infosehatku.club^$generichide
@@||memecenter.com/app/get_morecontent has been made redundant by @@||memecenter.com/app/
@@||readcomiconline.to/cdn-cgi$popup has been made redundant by @@||readcomiconline.to$popup
@@||reutersmedia.net/resources/r/?$image,domain=reuters.com has been made redundant by @@||reutersmedia.net/resources^
@@||watchcartoononline.io/inc/js/tabcontent.js has been made redundant by @@||watchcartoononline.io/inc/
@@||youmaker.com/js/prebid.js$domain=epochtimes.com has been made redundant by @@||youmaker.com/js/prebid.js
@@||innovid.com/crossdomain.xml$domain=channel4.com has been made redundant by @@/crossdomain.xml$domain=channel4.com
@@||media.fastclick.net/iab/verify$domain=youronlinechoices.com has been made redundant by @@||media.fastclick.net/iab^$domain=youronlinechoices.com
missoulian.com##div.ad-col has been made redundant by gazettetimes.com,missoulian.com,kenoshanews.com,cumberlink.com,trib.com,dailyprogress.com,stltoday.com,journalstar.com##.ad-col
porngun.net##.adv has been made redundant by porngun.net,eteenporn.com,bobs-tube.com,camwhores.tv,sexboomcams.com,pornformance.com,vivud.com##.adv
pornoxo.com##.video-player .video-extra-overlay has been made redundant by ashemaletube.com,pornoxo.com,boyfriendtv.com,fetishpapa.com##.video-extra-overlay
pornoxo.com##.videoDetail > div[class*="advi"] has been made redundant by pornoxo.com##div[class*="advi"]
pushsquare.com##.insert.with-label has been made redundant by pushsquare.com##.insert
tampermonkey.net###ads_are_visible+table div.adventing.house has been made redundant by tampermonkey.net##.adventing
uploadedpremiumlink.xyz##.col-md-12 > div[style^="margin-top:"][style*="width: 728px;height: 90px;"]:not([class]):not([id]) has been made redundant by uploadedpremiumlink.xyz##div[style*="width: 728px;height: 90px;"]
vg247.com##.page > .content > .low-leader-container has been made redundant by vg247.com##.content > .low-leader-container
vivud.com##.adv has been made redundant by porngun.net,eteenporn.com,bobs-tube.com,camwhores.tv,sexboomcams.com,pornformance.com,vivud.com##.adv
vivud.com##.in-gallery-banner has been made redundant by vivud.com##.in-gallery-banner
watchseries.cr##a[href^="http://www.xl415.com/apu.php"] has been made redundant by watchseries.cr##a[href*="/apu.php"]
whatismybrowser.com##.content > .fun has been made redundant by whatismybrowser.com##.fun
whatismybrowser.com##.fun.desktop has been made redundant by whatismybrowser.com##.fun
windowscentral.com##.header-top-alert-bar > .content > .surface-2018-order has been made redundant by androidcentral.com,windowscentral.com,imore.com##.header-top-alert-bar
wololo.net##.content > div[style="background-color:#DDEEDD;"]:not([class]):not([id]) has been made redundant by wololo.net##.content > div[style*="background-color"]:not([class]):not([id])
yts.ag##.rating-row.hidden-sm > a.fd has been made redundant by yts.ag##.hidden-sm.rating-row > a
yts.ag##.rating-row.hidden-sm > a[class^="button-green-download"][href="javascript:void(0);"] has been made redundant by yts.ag##.hidden-sm.rating-row > a
||adilk.ilikecomix.com/*.js$domain=xyzcomics.com has been made redundant by ||adilk.ilikecomix.com^
||ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js$domain=eurogamer.net has been made redundant by @@||ajax.googleapis.com/ajax/libs/jquery/
||api.camclips.cc/api/proxy?a= has been made redundant by ||api.camclips.cc/api/*?a=
||bestadbid.com/apu.php$domain=tusfiles.net has been made redundant by ||bestadbid.com^
||cdn.taboola.com/libtrc/cbsinteractive-downloadcom/loader.js$domain=cnet.com has been made redundant by @@||cdn.taboola.com/libtrc/*/loader.js$domain=cnet.com
||d13k7prax1yi04.cloudfront.net/midnight.jquery.min.js has been made redundant by ||d13k7prax1yi04.cloudfront.net^
Whitelisting for an identical blocklisting: quora.com#@#.PromptsList
Whitelisting for an identical blocklisting: pornhub.com,pornhub.org,pornhub.net,pornhubthbh7ap3u.onion#@#.sectionWrapper > div[class*=" "]:not([class="sectionTitle"]):not([class="reset"])
Whitelisting for an identical blocklisting: redtube.com#@#.removeAdLink
139.99.120.222##.navbar-brand#fbawah has been made redundant by 139.99.120.222##.navbar-brand
mp3.big.az##body > a[href="http://unvan.az"][target="_blank"] has been made redundant by big.az##a[href="http://unvan.az"]
mp3.xodver.az#?#.main_content_info > .main_tittle:contains(Reklam) + .main_in_content has been made redundant by mp3.xodver.az#?#.main_content_info > .main_tittle:contains(Reklam)
mp3.xodver.az#?#.main_right_side > .main_tittle:first-child:contains(Reklam) + .main_in_content has been made redundant by mp3.xodver.az#?#.main_right_side > .main_tittle:first-child:contains(Reklam)
novelasesp.blogspot.de,novelasesp.blogspot.com,novelasesp.blogspot.com.au,novelasesp.blogspot.pe###HTML2 has been made redundant by novelasesp.blogspot.de,novelasesp.blogspot.com,novelasesp.blogspot.com.au,novelasesp.blogspot.pe###HTML2
nudeteenwhore.com#?#.title:has(> h2:contains(Recommend)) + div.thumbs has been made redundant by nudeteenwhore.com#?#.title:has(> h2:contains(Recommend))
pilot.wp.pl#@#.ads has been made redundant by autokult.pl,fotoblogia.pl,komorkomania.pl,gadzetomania.pl,money.pl,so-magazyn.pl,o2.pl,pudelek.pl,wp.pl#@#.ads
pluspremieres.us#@##ad-container has been made redundant by pluspremieres.us#@##ad-container
pluspremieres.us#@#.adsbygoogle has been made redundant by pluspremieres.us#@#.adsbygoogle
pogoda.wp.pl#?#.grid-container > div[class^="grid-"] > .list > ul > li > div[class*=" "]:matches-css(background-image: /^url\(data:image/png;base64,iVBOR/):has(> div:only-child:empty) has been made redundant by pogoda.wp.pl#?#.grid-container > div[class*=" "]:matches-css(background-image: /^url\(data:image/png;base64,iVBOR/):has(> div:only-child:empty)
pogoda.wp.pl#?#.grid-container > div[class^="grid-"] > div[class*=" "]:matches-css(background-image: /^url\(data:image/png;base64,iVBOR/):has(> div:only-child:empty) has been made redundant by pogoda.wp.pl#?#.grid-container > div[class*=" "]:matches-css(background-image: /^url\(data:image/png;base64,iVBOR/):has(> div:only-child:empty)
pogoda.wp.pl#?#.grid-container > div[class^="grid-"] > div[class] > div[class*=" "]:matches-css(background-image: /^url\(data:image/png;base64,iVBOR/):has(> div:only-child:empty) has been made redundant by pogoda.wp.pl#?#.grid-container > div[class*=" "]:matches-css(background-image: /^url\(data:image/png;base64,iVBOR/):has(> div:only-child:empty)
qan.az#?#div[style="text-align: center"] > b > b[-ext-contains="Reklam:"]+a[href^="http://qan.az/chat/simaz/go.php?id="]+br has been made redundant by qan.az#?#div[style="text-align: center"] > b > b[-ext-contains="Reklam:"]
Whitelisting for an identical blocklisting: pcmag.com#@#iframe[title="3rd party ad content"]
Whitelisting for an identical blocklisting: vg247.com#@#.newsad
so-magazyn.pl,o2.pl,pudelek.pl,wp.pl#@#.ads has been made redundant by autokult.pl,fotoblogia.pl,komorkomania.pl,gadzetomania.pl,money.pl,so-magazyn.pl,o2.pl,pudelek.pl,wp.pl#@#.ads
so-magazyn.pl,o2.pl,pudelek.pl,wp.pl#@#[class^="advertisement"] has been made redundant by autokult.pl,fotoblogia.pl,komorkomania.pl,gadzetomania.pl,money.pl,so-magazyn.pl,o2.pl,pudelek.pl,wp.pl#@#[class^="advertisement"]
soulreaperzone.com##body > div[id^="ai-adb-"][style^="position: fixed; top:"][style*="z-index: 9999"] has been made redundant by jpopsingles.eu,nulledfree.pw,chuksguide.com,tencentgamingbuddy.co,gplcanyon.com,civildigital.com,paksociety.com,iptv4sat.com,apkmb.com,loveroms.online,loveroms.online,soulreaperzone.com,freecoursesonline.me##body > div[id^="ai-adb-"][style^="position: fixed; top:"][style*="z-index: 9999"]
thewatchcartoononline.tv#@#.adBanner has been made redundant by thewatchcartoononline.tv,wco.tv,wcostream.com,wcoanimesub.tv,wcoanimedub.tv#@#.adBanner
tinhte.vn##.root > header + div[class^="jsx-"].main > .root > div[class^="jsx-"].main > div[class^="jsx-"].main > div[style="width:320px;height:50px"]:empty has been made redundant by tinhte.vn##div[style="width:320px;height:50px"]
turizm.fins.az##.banner has been made redundant by fins.az##.banner
vg.no##.appnexus.ad has been made redundant by vg.no##.ad
www.filmweb.pl#@#.ws__wrapper has been made redundant by filmweb.pl#@#.ws__wrapper
xz2.xyz##+js(window.open-defuser) has been made redundant by xz2.xyz##+js(window.open-defuser)
||21moviemaniaasli.club/asset/default/player/plugins/vast-*.js has been made redundant by /asset/default/player/plugins/vast-*.js
||ads2.contentabc.com^$third-party has been made redundant by ||ads2.contentabc.com^$third-party
||alio.lt/info*.html$domain=delfi.lt has been made redundant by ||alio.lt^$domain=delfi.lt
||beglorena.com^$third-party has been made redundant by ||beglorena.com^
||bestadbid.com^$third-party has been made redundant by ||bestadbid.com^
||bfe4e6d364be199.com^$third-party has been made redundant by ||bfe4e6d364be199.com^
||bms.adjarabet.com^$third-party has been made redundant by ||adjarabet.com^$third-party
||bodelen.com^$third-party has been made redundant by ||bodelen.com^$third-party
||bp.blogspot.com^$image,domain=nontonmovie33.net has been made redundant by ||bp.blogspot.com^$image,domain=movie303.com|nontonmovie33.net
||bubblesmedia.ru^$third-party,popup has been made redundant by ||bubblesmedia.ru^$popup
||d1r90st78epsag.cloudfront.net^ has been made redundant by ||d1r90st78epsag.cloudfront.net^
||d2ho1n52p59mwv.cloudfront.net^ has been made redundant by ||d2ho1n52p59mwv.cloudfront.net^
||dd.70yst.com^$third-party has been made redundant by ||70yst.com^$third-party
||delfi.lv/news_export/widget_creator.php$domain=reklama.vesti.lv has been made redundant by ||delfi.lv/news_export$domain=~delfi.ee|~delfi.lv
||exe.io/files/p10.js has been made redundant by ||exe.io/files/p*.js
||flyordie.com/games/online/iframeafg*.html has been made redundant by @@||flyordie.com/games/online/iframeafg*.html
||givirsou.net^$third-party has been made redundant by ||givirsou.net^
||javporn.best/asset/default/player/plugins/vast-*.js has been made redundant by /asset/default/player/plugins/vast-*.js
||js.hdzog.com/saber/hdzog/dutti/ has been made redundant by ||js.hdzog.com/saber/
||kickassanime.io/codea/720x90*.html has been made redundant by ||kickassanime.io/codea/
||kurkizraka.com^$third-party has been made redundant by ||kurkizraka.com^
||niltutch.com^$third-party has been made redundant by ||niltutch.com^$third-party
||pegloang.com^ has been made redundant by ||pegloang.com^
||rotumal.com^$popup,domain=dailymotiontomp3.com has been made redundant by ||rotumal.com^$document,popup
||sexu.com/ifrm/*?ad=awn has been made redundant by ||sexu.com/ifrm/
||show.g.mediav.com^$third-party has been made redundant by ||mediav.com^$third-party
||superfastcdn.com^$third-party has been made redundant by ||superfastcdn.com^
||util.simply-hentai.com/prod/$script,image,domain=simply-hentai.com has been made redundant by ||util.simply-hentai.com/prod/
||velocecdn.com^$third-party has been made redundant by ||velocecdn.com^
||voastauz.net^$third-party has been made redundant by ||voastauz.net^
||waubasou.com^$third-party has been made redundant by ||waubasou.com^
||xcouj.com^$third-party has been made redundant by ||xcouj.com^$third-party
||xozilla.com/player/html.php?aid=pause_html&video_id=*&*referer= has been made redundant by ||xozilla.com/player/html.php
||zbporn.com/xdman/ has been made redundant by ||zbporn.*/xdman/
||zbporn.com/xdman/license.*.js has been made redundant by ||zbporn.*/xdman/
||zbporn.com/zpi/zp.js has been made redundant by ||zbporn.*/zpi/zp.js
```

* **Expected behaviour**: 

N/A

***Steps to reproduce the problem***:

N/A

***System configuration***

**Filters:**

![image](https://user-images.githubusercontent.com/22780683/94770484-fedfab80-03b4-11eb-9d07-8a25f679b118.png)

Additionally, for temporary Twitch purposes, I have "AdGuard Base filter (Optimized)" and no other lists turned on in AdGuard for Windows.

Information                            | Value
---                                    | ---
Operating system:                      | Windows 10
Operating system version (Android/iOS) | May 2020 Update
Browser:                               | Chrome 85.0.4183.121 x64
AdGuard version:                       | Nano Adblocker 1.0.0.153 + Nano Defender 15.0.0.205 + AdGuard for Windows 7.5.1 + IDCAC 3.2.2
Filters enabled:                       | See screenshot and notice above.
AdGuard mode (Android only):           | N/A
Filtering quality (Android only):      | N/A
HTTPS filtering (Android only):        | On
Simplified filters (iOS only)          | Off (Nano Adblocker) + On (AdGuard for Windows)
AdGuard DNS:                           | None
Stealth mode options (Windows only)    | N/A
Helpdesk ID (if exists):               | N/A

[//]: # (This template is meant for missed ad/false positive reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)

DandelionSprout committed:
Update specific.txt

DandelionSprout committed:
Update specific.txt

DandelionSprout committed:
Update antiadblock.txt

DandelionSprout committed:
Update antiadblock.txt

DandelionSprout committed:
Update antiadblock.txt

DandelionSprout committed:
Update specific.txt

DandelionSprout committed:
Update whitelist.txt

DandelionSprout committed:
Update specific.txt

DandelionSprout committed:
Update specific.txt

DandelionSprout committed:
Update adservers.txt

DandelionSprout committed:
Update whitelist.txt

Yuki2718 commented:
Great! I think at some point in near future re-evaluation of old scriptlet rules will be needed too.

Alex-302 commented on EnglishFilter/sections/adservers.txt:
```diff
@@ -2992,7 +2975,6 @@ www.llllllllllll.net^$third-party
 @@||amgload.net/z|$domain=sinoptik.ua|gismeteo.ua
 !
 !#if (adguard_ext_safari || adguard_app_ios || adguard_ext_android_cb)
-||analyticsq.com^
```
Not redundant. Please read directive above)

Alex-302 replied:
Reverted.

Alex-302 commented on EnglishFilter/sections/antiadblock.txt:
```diff
@@ -2639,6 +2594,7 @@ linksht.com#%#//scriptlet("prevent-setTimeout", "_$_")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/35273
 !+ NOT_PLATFORM(windows, mac, android)
 @@||theseotools.net^$generichide
+! https://github.com/AdguardTeam/AdguardFilters/issues/19880
```
wrong copy-pasta?

Alex-302 replied:
Removed.

Alex-302 commented on EnglishFilter/sections/antiadblock.txt:
```diff
@@ -4939,11 +4877,6 @@ rollercoin.com#%#AG_onLoad(function() { window.addBlockTest = function() {}; });
 @@||zonebourse.com/js/googleads.js
 ! https://github.com/AdguardTeam/AdguardFilters/issues/19889
 @@||rules.noadblock.org^$domain=hackintoshzone.com
-! https://github.com/AdguardTeam/AdguardFilters/issues/19880
-!+ PLATFORM(ext_ff, ext_opera, ios, ext_android_cb, ext_ublock)
-@@||singingdalong.blogspot.com^$generichide
-! https://github.com/AdguardTeam/AdguardFilters/issues/19837
-@@||imgadult.com/anex/alt2.js
```
Why these removed?
```
! https://github.com/AdguardTeam/AdguardFilters/issues/19880	
!+ PLATFORM(ext_ff, ext_opera, ios, ext_android_cb, ext_ublock)	
@@||singingdalong.blogspot.com^$generichide	
! https://github.com/AdguardTeam/AdguardFilters/issues/19837	
@@||imgadult.com/anex/alt2.js
```

Alex-302 replied:
It seems ok)

Alex-302 commented on EnglishFilter/sections/specific.txt:
```diff
@@ -10906,7 +10879,6 @@ turbobit.net##body > a#body-link
 !
 ! /expla*.js^$important,domain=rarbgto.org.
 ! /expla*.js^$important,domain=rarbgway.org|rarbgunblock.com|rarbgcore.org|rarbgp2p.org|rarbgenter.org|rarbg2019.org|rarbgunblocked.org|rarbgproxied.org|rarbgmirrored.org|rarbgprx.org|rarbgtor.org|rarbgget.org|proxyrarbg.org|rarbgto.org|rarbgunblock.org|rarbg2018.org|rarbg.bypassed.org|rarbg.is|rarbg.to|rarbg.unblockall.org|rarbg.unblocked.kim|rarbgaccess.org|rarbgmirror.com|rarbgmirror.org|rarbg ...
-/:\/\/rarbg.bypassed.org\/cdn\/static\/20\/js\/[a-z0-9]{20,}.js/$domain=rarbg.bypassed.org
```
better remove another copy

Alex-302 commented:
Please note that the Safari's API is working not like Chrome, and some rules, which looks like redundant - are still required(URL rules).

Alex-302 committed:
Update adservers.txt

reverted `||analyticsq.com^`  under `!#if (adguard_ext_safari || adguard_app_ios || adguard_ext_android_cb)`

Alex-302 committed:
Update antiadblock.txt

Removed wrong comment `! https://github.com/AdguardTeam/AdguardFilters/issues/19880`

Alex-302 commented:
Thanks!
